### ISSUE TITLE
formatURI not working for parsed playlist

### ISSUE THREAD

**Issue by zhaobenny** [TooTallNate/spotify-uri#6]
```
var parsed = spotifyUri.parse("https://open.spotify.com/playlist/37i9dQZF1DWVqfgj8NZEp1?si=rEPRsn4NRzCW4Pzj_8ivog");
var uri = spotifyUri.formatURI(parsed);
```
gives error 
```
..../node_modules/spotify-uri/formatURI.js:58
        return escape(str.replace(/ /g, '+'));
                          ^
TypeError: Cannot read property 'replace' of undefined
```

**Comment by miraclx** [comment]
Just stumbled on this error too

Seems @TooTallNate's logic only formats for users playlists
https://github.com/TooTallNate/spotify-uri/blob/dff69e92a13bca6d0e7292a0f3263e576a487fd2/formatURI.js#L26-L29

Opening a pull request...

**Comment by miraclx** [comment]
Until the author merges this pull request you could rebase here

``` bash
npm install git://github.com/miraclx/spotify-uri#patch-1
```

**Comment by TooTallNate** [comment]
Fixed in v1.1.0. Thanks @miraclx!
